portfolio/src/main/webapp/script.js

```diff
@@ -37,10 +37,13 @@ const getComments = () => {
   .then(comments => {
     console.log(comments);
     const container = document.getElementById('comments-container');
+    container.innerHTML = ''; // First Clear Children
     comments.forEach(comment => container.appendChild(addCommentElement(comment)));
   });
 };
 
+const initialiseCommentRefreshButton = () => document.getElementById("comments-refresh").onclick = getComments;
```

Comment:
[use single-quotes for js string literals](https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes)